### PR TITLE
Updated current accommodation to include extra questions

### PR DESCRIPTION
**Pull request by tony-gledhill**

### CODE REVIEW

File: components/application/application-forms.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export default function ApplicationForms({
   const activeStepId =
     formSteps.find((step) => step === activeStep) ?? formSteps[0];
   const formData = getFormData(activeStepId);
+  console.log('formData', formData);
```
Comment (tcmorris) [inline components/application/application-forms.tsx:42]: `console.log` to remove.

**Review by tcmorris — COMMENTED** [review]
In testing, the values didn't appear to be saving to the redux store so they weren't loading back after submission. (have found a fix)
The actual flow for the forms seemed to work ok.

File: components/application/application-forms.tsx (hunk 1 above)
Comment (tony-gledhill) [reply components/application/application-forms.tsx:42]: My bad. Good spot

**Comment by tony-gledhill** [comment]
The values of the first set of questions are being saved under current-accommodation but the others are being saved under current-accommodation-host-details and current-accommodation-landlord-details as the questions are being saved separate files. If we were able to have multiple conditions on the conditionalDisplay field, we could bring the data back into one file

**Pull request merged**